blackbox/docs/administration/ssl.txt

```diff
@@ -11,13 +11,12 @@ Secured Communications (SSL/TLS)
 Secured communication allows you to encrypt traffic between the CrateDB node
 and a client. This applies to connections using HTTP (i.e. Admin UI, Crash, Rest),
 connections using the PostgreSQL Wire Protocol (i.e. JDBC, psql), and MQTT 
-connections (i.e. MQTT ingestion source)
+(i.e. :ref:`MQTT ingestion source <ingest_mqtt>`)
```

Comment:
Minor refinement to remove another "connections":

> This applies to connections using HTTP (i.e. Admin UI, Crash, Rest),
~~connections using~~ the PostgreSQL Wire Protocol (i.e. JDBC, psql), and MQTT
(i.e. :ref:`MQTT ingestion source <ingest_mqtt>`)**.**